src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,34 @@
 package mate.controller;
 
+import static mate.dao.MyCoolResource.openResource;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
+@WebServlet(urlPatterns = "/index")
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        myResource.write(LocalDateTime.now().toString());
+        req.getRequestDispatcher("WEB-INF/views/index.jsp").forward(req,resp);
+    }
+
+    @Override
+    public void init() throws ServletException {
+        myResource = openResource();
```

Comment:
```suggestion
        myResource = MyCoolResource.openResource();
```
will be more informative (it's optional)